lib/openapi/revisions/fps-spec-1.5.0-20231130.json

```diff
@@ -0,0 +1,870 @@
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/AllVisitsResponse"
+                }
+              }
+            }
+          },
+          "400": {
+            "description": "Bad request"
+          },
+          "401": {
+            "description": "Authorization failure due to invalid authentication credentials."
+          }
+        }
+      }
+    },
+    "/api/ds/v1/metrics": {
+      "get": {
+        "summary": "Retrieve metrics data",
+        "description": "Returns metrics data for a given time period. If the data for a particular period has a value of 'null', it denotes that the metrics could not be calculcated for that time period. For example accuracy cannot be calculated if there was no cookied visit for that day.",
+        "operationId": "getMetricsData",
+        "tags": [
+          "Metrics"
+        ],
+        "security": [
+          {
+            "ApiKeyAuth": []
+          }
+        ],
+        "parameters": [
+          {
+            "name": "start",
+            "in": "query",
+            "description": "The start date used for querying metrics data. Must be provided in RFC3339 date-time format and UTC.",
+            "required": false,
+            "example": "YYYY-MM-DDTHH:MM:SSZ",
+            "schema": {
+              "type": "string",
+              "format": "date-time"
+            }
+          },
+          {
+            "name": "end",
+            "in": "query",
+            "description": "The end date used for querying metrics data. Must be provided in RFC3339 date-time format and UTC.",
+            "required": false,
+            "example": "YYYY-MM-DDTHH:MM:SSZ",
+            "schema": {
+              "type": "string",
+              "format": "date-time"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully retrieved metrics data.",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/MetricsResponse"
+                }
+              }
+            }
+          },
+          "400": {
+            "description": "Bad request"
+          },
+          "401": {
+            "description": "Authorization failure due to invalid authentication credentials."
+          }
+        }
+      }
+    },
+    "/api/ds/v1/metrics/topn/visits": {
+      "get": {
+        "summary": "Retrieve top visits count with their visitor IDs.",
+        "description": "Returns top visits count with their visitor IDs for a given time period.",
+        "operationId": "getTopNVisitsData",
+        "tags": [
+          "Metrics"
+        ],
+        "security": [
+          {
+            "ApiKeyAuth": []
+          }
+        ],
+        "parameters": [
+          {
+            "name": "n",
+            "in": "query",
+            "description": "Number of top visits data to return to.",
```

Comment:
I do not know what this description means.

Reply:
@joubert-macrometa and @rahulprabhune-macrometa, please explain this.

Reply:
```suggestion
            "description": " Number of visitors.",
```

Reply:
@diana-macrometa I think this is better.  But I am not sure that does the job.  

The api returns the top N number of visitors.  This parameter is what you put in for N.  I could not come up with a better way of saying this.

Reply:
Is it the maximum number of visitors in a given time window?